lib/project_types/script/layers/application/build_script.rb

```diff
@@ -6,6 +6,10 @@ module Application
       class BuildScript
         class << self
           def call(ctx:, task_runner:, script_project:)
+            extension_point = ExtensionPoints.get(type: script_project.extension_point_type)
+            # TODO: Replace with with the sdks.for method from the sparse_checkout PR
+            library_name = extension_point.sdks.all.find { |ep| ep.class.language == script_project.language }.package
```

Comment:
Nit: If we're referring to these as `libraries`, then I think we should also rename `.package` to `.library_name`. At first I was a little confused at what this line was doing.

Reply:
👍  We need to do a passthrough of the code to update the terminology I think. 
In this case, since we need to revisit this line after the sparse PR is merged (to use the `sdks.for` from there), I'd like to leave it as is.